GeneratorInterface/LHEInterface/src/LHEReader.cc

```diff
@@ -47,6 +47,7 @@ namespace lhef {
   class LHEReader::FileSource : public LHEReader::Source {
   public:
     FileSource(const std::string &fileURL) {
+      using namespace edm::storage;
       auto storage = StorageFactory::get()->open(fileURL, IOFlags::OpenRead);
```

Comment:
Hi @Dr15Jones : Shouldn't it be edm::Storage::StorageFactory in this line according to our coding rules?

Reply:
I've been unable to determine which coding rule to which you refer. Could you point me to the reference?

Reply:
I thought I got that comment once but indeed I cannot spot in the rules. Let me sign.